IBX-6116: Scenario block added in 'Block reference' page (#208)
ezsystems/user-documentation: docs/content_management/block_reference.md

After submitting the settings, all images in the folder will appear in the Gallery block.
Note that selecting a folder containing Content items other than images results in displaying only a link to the folder they are stored in.

## Ibexa Connect block

> On the **Basic** tab, set values in the following fields:

Reviewer: ```suggestion
In the **Basic** tab, set values in the following fields:
```

Author: Let's keep "on" for the page consistency.

- **Name** – Enter a name for the Page block.
- **Webhook link** - Enter the link for the Ibexa Connect webhook.

> On the **Design** tab, in the **View** field, select the template be used to present the webhook.

Reviewer: ```suggestion
In the **Design** tab, in the **View** field, select the template for the webhook.
```

Author: Let's keep "on" for the page consistency.
